run_mac.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+THIS_DIR="$(pwd)"
+ROS_DIR="$THIS_DIR/ros"
+
+# Re-build the code
+echo "Building the code..."
+$THIS_DIR/build.sh
+
+# Launch simulator if it's not running yet
+if pgrep "mac_sys_int" > /dev/null
```

Comment:
Oops, I didn't know they have a different process name. Then Iagree it makes sense to have 2 separate scripts.